src/number.rs

```diff
@@ -18,7 +18,7 @@ use error::ErrorCode;
 #[cfg(feature = "arbitrary_precision")]
 /// Not public API. Should be pub(crate).
 #[doc(hidden)]
-pub const TOKEN: &'static str = "$serde_json::private::Number";
+pub const TOKEN: &str = "$serde_json::private::Number";
```

Comment:
In the future it would be better to make this kind of unrelated change in a separate commit in a separate PR. I've moved it to 57bc9ae77b31969ee6888899ca39f3e86ff0890e.